tests/unittest/auto_deploy/singlegpu/smoke/test_ad_speculative_decoding.py

```
@@ -92,6 +92,70 @@ def test_super_mtp_smoke():
+    """Test one-model MTP with flashinfer_ssm + ssm_replay=True.
+
+    Uses mamba_head_dim=64 (required by FlashInfer) and ssm_state_size=64
+    (>= 16 for the replay kernel's tl.dot constraint), actually executing
+    _replay_precompute_kernel and _replay_state_update_kernel.
+    """
+    test_prompt = "What is the capital of France?"
+    model_hub_id = "nvidia/NVIDIA-Nemotron-3-Super-120B-A12B-BF16"
+    model_path = hf_id_to_local_model_dir(model_hub_id)
+
+    # Get base small-model config and update SSM dims for FlashInfer + replay.
+    # hidden_size must equal mamba_num_heads × mamba_head_dim (4 × 64 = 256).
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)